examples/config/ci/deployer.json

```diff
@@ -0,0 +1,153 @@
+{
+  "AWSProfile": "",
+  "AWSRegion": "us-east-1",
+  "AWSAvailabilityZone": "",
+  "AWSAMI": "ami-003d3d03cfe1b0468",
+  "ClusterName": "undefined",
+  "ClusterVpcID": "",
+  "AppInstanceCount": 2,
+  "AppInstanceType": "c7i.large",
+  "MetricsInstanceType": "t3.xlarge",
+  "AgentInstanceCount": 6,
+  "AgentInstanceType": "c7i.xlarge",
+  "ElasticSearchSettings": {
+    "InstanceCount": 0,
+    "InstanceType": "r6g.large.search",
+    "Version": "Elasticsearch_7.10",
+    "CreateRole": false,
+    "SnapshotRepository": "",
+    "SnapshotName": "",
+    "RestoreTimeoutMinutes": 45,
+    "ClusterTimeoutMinutes": 45
+  },
+  "RedisSettings": {
+    "Enabled": false,
+    "NodeType": "cache.m7g.2xlarge",
+    "ParameterGroupName": "default.redis7",
+    "EngineVersion": "7.1"
+  },
+  "JobServerSettings":{
+    "InstanceCount": 0,
+    "InstanceType": "c7i.xlarge"
+  },
+  "EnableAgentFullLogs": true,
+  "ProxyInstanceCount": 1,
+  "ProxyInstanceType": "c7i.xlarge",
+  "SSHPublicKey": "/home/runner/.ssh/mattermost-load-test.pub",
+  "TerraformStateDir" : "/home/runner/ltstatus",
+  "S3BucketDumpURI" : "",
+  "DBDumpURI": "",
+  "SiteURL": "",
+  "EnableNetPeekMetrics": false,
+  "TerraformDBSettings": {
+    "InstanceCount": 2,
+    "InstanceEngine": "aurora-postgresql",
+    "InstanceType": "db.r7g.xlarge",
+    "UserName": "mmuser",
+    "Password": "mostest80098bigpass_",
+    "EnablePerformanceInsights": true,
+    "DBParameters": [],
+    "ClusterIdentifier": ""
+  },
+  "ExternalDBSettings": {
+    "DriverName": "cockroach",
+    "DataSource": "",
+    "DataSourceReplicas": [],
+    "DataSourceSearchReplicas": []
+  },
+  "ExternalBucketSettings": {
+    "AmazonS3AccessKeyId": "",
+    "AmazonS3SecretAccessKey": "",
+    "AmazonS3Bucket": "",
+    "AmazonS3PathPrefix": "",
+    "AmazonS3Region": "us-east-1",
+    "AmazonS3Endpoint": "s3.amazonaws.com",
+    "AmazonS3SSL": true,
+    "AmazonS3SignV2": false,
+    "AmazonS3SSE": false
+  },
+  "ExternalAuthProviderSettings": {
+    "Enabled": false,
+    "KeycloakAdminUser": "mmadmin",
+    "KeycloakAdminPassword": "mmpass",
+    "KeycloakRealmFilePath": "",
+    "KeycloakDBDumpURI": "",
+    "GenerateUsersCount": 0
+  },
+  "MattermostDownloadURL": "https://latest.mattermost.com/mattermost-enterprise-linux",
+  "MattermostLicenseFile": "/home/runner/mattermost-load-test",
+  "MattermostConfigPatchFile": "",
+  "AdminEmail": "sysadmin@sample.mattermost.com",
+  "AdminUsername": "sysadmin",
+  "AdminPassword": "Sys@dmin-sample1",
+  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.23.0-rc1/mattermost-load-test-ng-v1.23.0-rc1-linux-amd64.tar.gz",
```

Comment:
Will this need constant updating?

Reply:
This set of config will need maintaining from our side, yes. I can add a step in the release process to review this. Also, I need to change this to the final release, not the RC-1.